test/node/server-broadcast-event.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application');
+
+const application = new jstp.Application(app.name, app.interfaces);
+const serverConfig =
+  { applications: [application], authPolicy: app.authCallback };
+
+let server;
+let port;
+
+test.beforeEach((done) => {
+  server = jstp.net.createServer(serverConfig);
+  server.listen(0, () => {
+    port = server.address().port;
+    done();
+  });
+});
+
+test.afterEach((done) => {
+  server.close();
+  done();
+});
+
+test.test('must broadcast event to all connected clients', (test) => {
+  const iface = 'someService';
+  const eventName = 'broadcast';
+  const eventArgs = ['hello'];
+
+  test.plan(12);
+
+  let connectionCount = 3;
+  const callback = () => {
+    if (--connectionCount === 0) {
+      server.broadcast(iface, eventName, ...eventArgs);
+    }
+  };
+
+  connectAndCheckEvent(test, iface, eventName, eventArgs, callback);
```

Comment:
You can use for loop here with `connectionCount` iterations, this will make this test more flexible.